apex/ProductConfiguration.apex

```diff
@@ -137,32 +137,58 @@ AttributeDefinition offerValidPeriod =
 insert offerValidPeriod;
 System.debug(offerValidPeriod);
 
+AttributeDefinition loanAmount =
+    new AttributeDefinition(Name = 'loanAmount', DataType='Currency', Description='LoanAmount', Label= 'LoanAmount', isActive=True);
+
+insert loanAmount;
+
+System.debug(loanAmount);
```

Comment:
Someone looking at the log file may not know what it means that loan amount is being logged.  Is it possible to do something like this
```
System.debug("Created attribute definition loanAmount: " + loanAmount);
```

Same in a few other places below.

Reply:
@SiemelNaran I have updated debug statements. Can you please take a look now